packages/ui/src/views/connectors/utils.tsx

```diff
@@ -0,0 +1,28 @@
+import { Icon } from '@components/icon'
+import { Logo } from '@components/logo/logo'
+import { SimpleIcon } from 'simple-icons'
+
+import { ConnectorConfigType } from './types'
+
+export const mapConnectorTypeToLogoName = (type: ConnectorConfigType): string | undefined => {
+  switch (type) {
+    case 'Github':
+      return 'github'
+    case 'Gitlab':
+      return 'gitlab'
+    case 'Bitbucket':
+      return 'bitbucket'
+    case 'Jira':
+      return 'jira'
+    case 'K8sCluster':
+      return 'kubernetes'
```

Comment:
These logo names `github`, `gitlab`, etc. should ideally come from an enum or a union type but I couldn't figure out how to dynamically generate it based on only the icons we're importing from `simple-icons`. But I will give it a try in a follow up PR.